.github/workflows/feg-workflow.yml

```diff
@@ -116,7 +131,7 @@ jobs:
       # Notify ci channel when failing
       # Plugin info: https://github.com/marketplace/actions/slack-notify
       - name: Notify failure to slack
-        if: ( steps.feg-precommit.outcome=='failure' || steps.feg-precommit-upload.outcome=='failure' ) && github.event_name == 'push'
+        if: ( steps.feg-precommit.outcome=='failure' || steps.feg-precommit-upload.outcome=='failure' || steps.radius-src-unit-tests.outcome=='failure' || steps.radius-lib-unit-tests.outcome=='failure') && github.event_name == 'push'
```

Comment:
Can we split this line up please?

Reply:
Use:
```
if: |
  condition \
 ...

Reply:
Done. Looking at the other places this syntax is used, it seems the ` \`s aren't needed, but please say if I'm wrong

Reply:
You are right actually, sorry for the confusion. They are only needed in Bash.